internal/gen/input.go

```diff
@@ -425,6 +432,7 @@ func (dg *dataGenerator) Do(f func(tbl flux.Table) error) error {
 			if err := f(tbl); err != nil {
 				return err
 			}
+			<-done
```

Comment:
This won't allow the user to cancel a long-running query, right? Is it a lot of trouble to fix it?

Reply:
I forgot about that. I'll update this.